x/bounty/client/cli/tx.go

```diff
@@ -131,3 +132,53 @@ func NewCreateProgramCmd() *cobra.Command {
 
 	return cmd
 }
+
+func NewSubmitFindingCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "submit-finding",
+		Short: "submit finding for a program",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			submitAddr := clientCtx.GetFromAddress()
+
+			desc, _ := cmd.Flags().GetString(FlagFindingDesc)
+			title, _ := cmd.Flags().GetString(FlagFindingTitle)
+
+			pid, err := cmd.Flags().GetUint64(FlagProgramID)
+			if err != nil {
+				return err
+			}
+			severityLevel, _ := cmd.Flags().GetInt32(FlagFindingSeverityLevel)
+			poc, _ := cmd.Flags().GetString(FlagFindingPoc)
+
+			msg, err := types.NewMsgSubmitFinding(
+				submitAddr.String(),
+				title,
+				desc,
+				pid,
+				severityLevel,
+				poc,
+			)
+			if err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(FlagFindingDesc, "", "The finding description")
```

Comment:
To  be consistent with the following lines, change to "The finding's description" instead?

Reply:
sure